### ISSUE TITLE
Slider stuck when value is set

### ISSUE THREAD

**Issue by dakkafex** [react-native-elements/react-native-elements#2515]
**Update:** The slider is not actually stuck but the button initially goes to the start, if you try to slide from where it should be, it does slide and stays to its proper position.

**Explain what you did (Required)**

useing the slider component with a value from state

**Expected behavior (Required)**

being able to slide

**Describe the bug (Required)**

slider is stuck at minimum position and wont respond to input

**To Reproduce (Required)**

Unable to reproduce on snack
but here it is how i have implemented it
https://snack.expo.io/PBHVS_RlT

**Screenshots (Required)**

![image](https://user-images.githubusercontent.com/7563763/90989495-717d9000-e59a-11ea-8495-5c05989ffcda.png)

**Your Environment (Required):**

| software              | version |
| --------------------- | ------- |
| react-native-elements |     2.0.4    |
| react-native          |     0.61.5    |

**Comment by dakkafex** [comment]
Currently i implemented a ghettofix by updating the value of the slider with an `onLayout` by 0.1 and using `Math.floor(sliderValue)` to force the thumb to move to the correct spot :P

**Comment by Elijen** [comment]
Same issue here. I used patch-package for a hotfix. Here's the patch:

`react-native-elements+2.3.1.patch`
```patch
diff --git a/node_modules/react-native-elements/src/slider/Slider.js b/node_modules/react-native-elements/src/slider/Slider.js
index 8deb5c2..fd95254 100644
--- a/node_modules/react-native-elements/src/slider/Slider.js
+++ b/node_modules/react-native-elements/src/slider/Slider.js
@@ -77,16 +77,18 @@ class Slider extends React.Component {
       onPanResponderTerminate: this.handlePanResponderEnd.bind(this),
     });
     this.isVertical = props.orientation === 'vertical';
+
+    this.componentDidUpdate(props, true);
   }
 
-  componentDidUpdate(prevProps) {
+  componentDidUpdate(prevProps, force) {
     const newValue = getBoundedValue(
       this.props.value,
       this.props.maximumValue,
       this.props.minimumValue
     );
 
-    if (prevProps.value !== newValue) {
+    if (prevProps.value !== newValue || force) {
       if (this.props.animateTransitions) {
         this.setCurrentValueAnimated(newValue);
       } else {

```

**Comment by ivansenic** [comment]
Any reason the patch was not included in the version `2.3.2`? Updating to this version makes all sliders in the app unusable.. It's really inside that this has not been fixed since the `2.0.4` :cry:

**Comment by pranshuchittora** [comment]
Hi @ivansenic 
We are trying our best to fix high priority bugs and ship some new features to make the RNE more stable and robust. The following issue will be fixed in the next release for sure  :smile:
